release-notes/opensearch-knn.release-notes-2.6.0.0.md

```diff
@@ -0,0 +1,30 @@
+## Version 2.6.0.0 Release Notes
+
+Compatible with OpenSearch 2.6.0
+
+### Enhancements
+
+* Adding p99.9, p100 and num_of_segments metrics to perf-tool ([#739](https://github.com/opensearch-project/k-NN/pull/739))
```

Comment:
Not sure perf tool change counts as enhancement. I think enhancements refer to core functionality related PRs.

Reply:
What should be the right category?

Reply:
Infrastructure

Reply:
Also, to be consistent with tense, can we change Adding -> Add

Reply:
done